content/en/docs/reference/glossary/event.md

```diff
@@ -4,13 +4,13 @@ id: event
 date: 2022-01-16
 full_link: /docs/reference/kubernetes-api/cluster-resources/event-v1/
 short_description: >
-   A report of an event somewhere in the cluster. It generally denotes some state change in the system.
+   Refers to an object that describes a notable occurrence or change of state in the system. 
```

Comment:
```suggestion
   Refers to an object that describes a notable occurrence or change of state in the system.
```
Refers what? Also, I'm not sure why we're adding an extra space. Can you explain if this was a typo?

Reply:
Yes, I am have added a space while typing mb.

Reply:
> Refers what? Also, I'm not sure why we're adding an extra space. Can you explain if this was a typo?

"refers to an object" is not the correct way to put it?